src/tests/EdFi.DataManagementService.Tests.E2E/appsettings.json

```diff
@@ -1,25 +1,22 @@
 {
     "UseTestContainers": true,
     "Serilog": {
-        "Using": ["Serilog.Sinks.File", "Serilog.Sinks.Console"],
+        "Using": [ "Serilog.Sinks.File", "Serilog.Sinks.Console" ],
         "MinimumLevel": {
-            "Default": "Information"
+            "Default": "Verbose"
         },
         "WriteTo": [
+            { "Name": "Console" },
+            { "Name": "Debug" },
```

Comment:
@simpat-adam 
It looks like "Debug" will log the details to the "Debug" output window. This is never a requirement for any of the other applications, though.